public/app/features/dashboard/components/LibraryPanelCard/LibraryPanelCard.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { useTheme, Icon, IconButton, stylesFactory, TagList, ConfirmModal } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export interface LibraryPanelCardProps {
+  id: number;
+  uid: string;
+  model: any;
+  title: string;
+  connectedDashboards: number[];
+  varCount: number;
+  lastEdited?: string;
+  lastAuthor?: string;
+  avatarUrl?: string;
+  onClick?: () => void;
+  onDelete?: () => void;
+}
+
+export const LibraryPanelCard: React.FC<LibraryPanelCardProps> = ({
+  title,
+  connectedDashboards,
+  varCount,
+  lastEdited,
+  lastAuthor,
+  children,
+  onClick,
+  onDelete,
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const [showDeletionModal, setShowDeletionModal] = useState(false);
+
+  const onDeletePanel = () => {
+    onDelete?.();
+    setShowDeletionModal(false);
+  };
+
+  return (
+    <>
+      <div className={cx(styles.panelCard)}>
```

Comment:
Please use cx only when composing class names:)

Reply:
We should use the Card component here (and extend it where needed)